doc/source/structures/vessels/vessel.rst

```diff
@@ -287,12 +272,6 @@ All vessels share a structure. To get a variable referring to any vessel you can
     use degrees here.  (That an backward compatibility for old scripts.
     It's been like this for quite a while.).
 
-    .. note::
-
-        .. versionchanged:: 0.15.4
-
-            This has been changed to a vector, as it should have been all along.
-
```

Comment:
Agree.  Although this change can cause an old script to fail (because it will be trying to use the return value which is now a vector like it was a scalar), the person looking for why in the docs will still find this suffix, which is now described as a vector, so the reason for the fail will still be apparent even if we remove this block.  It just won't be apparent that it used to be valid.  But it will be apparent what has to be done to fix it, so I"m okay with that.